INSTALLATION.md

```diff
@@ -1,26 +1,28 @@
-# Installing the Justice40 Tool
+# Justice40 Tool Installation
 
 *[¡Lea esto en español!](INSTALLATION-es.md)*
 
-## Required Software
-You'll need the following set up in your environment before continuing to run the Justice40 Tool code.
+This page documents the installation steps for some of the software needed to work with this project. After following the steps on this page, continue on to one of the following, depending on what you are trying to do:
 
-- git, node, and an IDE
+- If you want to quickly run everything locally, see [`QUICKSTART.md`](QUICKSTART.md).
```

Comment:
Probably worth calling this out more and saying STOP HERE if you don't want to go beyond getting the tool up and running fast.

Reply:
Done.